apps/src/templates/instructions/InstructionsCSF.jsx

```diff
@@ -376,13 +383,13 @@ const styles = {
   },
   leftCol: {
     position: 'absolute',
-    bottom: HEADER_HEIGHT + RESIZER_HEIGHT,
+    bottom: 13,
```

Comment:
why don't we need to calculate based on these heights anymore?

Reply:
+1 if 13 is a meaningful value it'd be helpful to make it a named constant so the intention is clear

Reply:
Good call, I'll update this to `RESIZER_HEIGHT`.  We no longer need to include `HEADER_HEIGHT` because it was compensating for an ancestor element that was unnecessarily oversized and bled over the header of the code area below.